### PR TITLE
Updated cluster loader docs per Bug 1761925

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1761925

Preview build: http://file.rdu.redhat.com/~ahardin/11262019/clusterloader-update/scalability_and_performance/using-cluster-loader.html

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@sjug Here are the updates as discussed. PTAL. If this looks okay, I can make changes for 4.1 and 4.2 as well, assuming that the version number is the only thing that would need adjusted for each version. Thanks!

**Comment by ahardin-rh** [comment]
@sjug is looking into some questions regarding auth. :mag_right:

**Comment by ahardin-rh** [comment]
@sjug Any updates here? Is there someone else I should follow up with? Thanks!

**Comment by ahardin-rh** [comment]
@sjug Hey, Sebastian. Any updates here?

**Comment by ahardin-rh** [comment]
@mffiedler Can you please provide QE review?

**Comment by ahardin-rh** [comment]
@wabouhamad Can you please provide QE review?

File: modules/running-cluster-loader.adoc [hunk 1]
```diff
@@ -5,14 +5,19 @@
 [id="running_cluster_loader_{context}"]
 = Running Cluster Loader
 
+.Prerequisites
+
+* The repository needs authentication. Use your existing authentication
```
Comment (jboxman) [inline modules/running-cluster-loader.adoc]: My only question would be, what does it mean for a repository to need authentication?

**Review by jboxman — APPROVED** [review]
Just one question, otherwise LGTM.

**Pull request merged**